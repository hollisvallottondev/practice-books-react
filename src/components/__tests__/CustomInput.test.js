import React from 'react';
import CustomInput from '../CustomInput/index.jsx';
import renderer from 'react-test-renderer';

test('CustomInput renders correctly', () => {
	const handleSearch = jest.fn(() => console.log('Handle search'));

	const component = renderer.create(
		<CustomInput 
			handleSearch={handleSearch} 
			icon={'fas fa-search'}
			placeHolder={'Find a book by title or author'}
		/>
	)

	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});