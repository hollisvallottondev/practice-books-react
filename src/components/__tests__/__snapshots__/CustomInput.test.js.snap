// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomInput renders correctly 1`] = `
<div>
  <div>
    <i
      className="fas fa-search"
    />
    <input
      onChange={[Function]}
      placeholder="Find a book by title or author"
      type="text"
      value=""
    />
  </div>
</div>
`;
